threeD-components/Trade.js

```diff
@@ -0,0 +1,303 @@
+  }
+  #threeD-trade-peers-page {
+    display: inline;
+    font-size: 80px;
+  }
+  </style>
+    <div class="threeD-trade">
+      <div class="threeD-trade-header">
+        <a class="threeD-trade-header-backBtn" id="threeD-trade-header-backBtn" onclick="threeD-trade-header-backBtn">
+          < Back
+        </a>
+        <h1>Trade</h1>
+      </div>
+      <div class="threeD-trade-content">
+        <h1 class="threeD-trade-sectionHeader">
+          Select Item:
+          <div class="threeD-trade-inventory-pagination">
+            <a id="threeD-trade-inventory-back"><</a>
+            <p id="threeD-trade-inventory-page">${props.trade.inventoryPage}</p>
+            <a id="threeD-trade-inventory-forward">></a>
+          </div>
+        </h1>
+        <div class="threeD-trade-inventory">
+          ${
+            inventoryItems.map((value, index) => {
+              return InventoryCard({
+                  id: value.id,
+                  name: value.filename,
+                  preview: value.preview,
+                  selected: selectedItem?.id === value.id,
+              })
+            }).join('')
+          }
+        </div>
+        <h1 class="threeD-trade-sectionHeader">
+          Select Peer:
+          <div class="threeD-trade-peers-pagination">
+            <a id="threeD-trade-peers-back"><</a>
+            <p id="threeD-trade-peers-page">${props.trade.peersPage}</p>
+            <a id="threeD-trade-peers-forward">></a>
+          </div>
+        </h1>
+        <div class="threeD-trade-peers">
+          ${
+            peers.map((value, index) => {
+              return PeerCard({
+                peerAddress: value.address,
+                selected: value.address === toPeer,
+              });
+            }).join('')
+          }
+        </div>
+        <div class="threeD-trade-actions">  
+          <div class="threeD-trade-info">
+            <h1 class="threeD-trade-info-header">
+              Selected Peer: 
+              <p class="threeD-trade-info-detail">${toPeer}</p>
+            </h1>
+            <h1 class="threeD-trade-info-header">
+              Item for Trade: 
+              <p class="threeD-trade-info-detail">${selectedName}</p>
+            </h1>
+          </div>
+          <div class="threeD-trade-actions-agreement"> 
+            <input style="cursor: pointer;" type="checkbox" id="threeD-trade-agreement" name="threeD-trade-agreement" ${agreement ? 'checked' : ''} onclick="threeD-trade-agreement">
+            <label style="font-size: 20px; cursor: pointer;" for="threeD-trade-agreement" id="threeD-trade-agreement-label"> I agree to trade my token.</label><br>
+            <div class="threeD-spacer"></div>
+            <a class="threeD-trade-cancel" id="threeD-trade-cancel" onclick="threeD-trade-cancel">Cancel</a>
+            <a class="threeD-trade-accept ${!agreement || !toPeer || !fromPeer || !selectedItem ? 'disabled' : ''}" id="threeD-trade-accept" onclick="threeD-trade-accept">Accept</a>
```

Comment:
This could be abstracted into a local function the names what logic we are trying to capture in this condition.

Reply:
https://github.com/webaverse/app/pull/160/commits/be30d4518b2826fc5f7c3d6fc00120ba9cca4c9a